bee-message/src/output/alias.rs

```diff
@@ -10,7 +10,14 @@ use crate::{
     Error,
 };
 
-use bee_common::packable::{Packable, Read, Write};
+use bee_packable::{
+    bounded::BoundedU32,
+    error::{UnpackError, UnpackErrorExt},
+    packer::Packer,
+    prefix::BoxedSlicePrefix,
+    unpacker::Unpacker,
+    Packable,
+};
 
 ///
```

Comment:
This was already here before, but why is this comment blank?

Reply:
@thibault-martinez ?

Reply:
Because I will do a PR with documentation soon, I put this to make it compile so that it wasn't blocking to merge